MiniApp/Classes/core/JavascriptBridge/Chat/ChatMessageBridgeDelegate.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public protocol ChatMessageBridgeDelegate: class {
+    /// Triggered when Mini App wants to send a message to a contact.
+    /// Should open a contact chooser which allows the user to choose a single contact,
+    /// and should then send the message to the chosen contact.
+    /// Should invoke completionHandler success with the ID of the contact which was sent the message.
+    /// If the user cancelled sending the message, should invoke completionHandler success with nil value.
+    /// Should invoke completionHandler error when there was an error.
+    func sendMessageToContact(_ message: MessageToContact, completionHandler: @escaping (Result<String?, MASDKError>) -> Void)
+
+    /// Triggered when Mini App wants to send a message to a specific contact.
+    /// Should send a message to the specified contactId without any prompt to the User.
+    /// Should invoke completionHandler success with the ID of the contact which was sent the message.
+    /// If the user cancelled sending the message, should invoke completionHandler success with nil value.
+    /// Should invoke completionHandler error when there was an error.
+    func sendMessageToContact(_ contactId: String, message: MessageToContact, completionHandler: @escaping (Result<String?, MASDKError>) -> Void)
```

Comment:
Shouldn't this be changed to `sendMessageToContactId` ??

Reply:
That was the spec I could find in ticket MINI-2474 or on Confluence. I think it's better we keep it like this or Android may also have to update their code

Reply:
I think Android is already using `sendMessageToContactId`, so we should change this to match both Android and JS. Sorry, that's the problem with specs - they're always outdated.

Reply:
That's fine @corycaywood , I think I did not pay enough attention :)